.github/workflows/docker-promote.yml

```diff
@@ -58,24 +58,49 @@ jobs:
           cache-disabled: true
 
       - name: Docker upload
-        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_NAME }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" dockerUploadRelease
+        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_VERSION }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" dockerUploadRelease
 
       - name: Docker manifest
-        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_NAME }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" manifestDockerRelease
+        run: ./gradlew "-Prelease.releaseVersion=${{ env.RELEASE_VERSION }}" "-PdockerOrgName=${{ env.registry }}/${{ secrets.DOCKER_ORG }}" manifestDockerRelease
 
   docker-verify:
-    needs: [validate, docker-promote]
+    needs: [validate,docker-promote]
     env:
-      RELEASE_NAME: ${{ needs.validate.outputs.release_name }}  # Use the output from the pre_process_release job
-    runs-on: ubuntu-22.04
-    permissions:
-      contents: read
-      actions: write
+      CONTAINER_NAME: besu-check
+      RELEASE_VERSION: ${{ needs.validate.outputs.release_version }}
+    runs-on: ${{ matrix.combination.runner }}
+    timeout-minutes: 4
+    strategy:
+      matrix:
+        combination:
+          - tag: ${{ needs.validate.outputs.release_version }}
+            platform: ''
+            runner: ubuntu-22.04
+          - tag: ${{ needs.validate.outputs.release_version }}-amd64
+            platform: 'linux/amd64'
+            runner: ubuntu-22.04
+          - tag: latest
+            platform: ''
+            runner: ubuntu-22.04
+          - tag: ${{ needs.validate.outputs.release_version }}-arm64
```

Comment:
Haven't we already checked the release_version variants in draft-release?

Think we should check latest, latest-amd64, latest-arm64 here

Reply:
Good point. I will add latest-amd64 and latest-arm64